packages/stg-evm-usdt-v2/test/tetherToken.test.js

```diff
@@ -0,0 +1,181 @@
+/* eslint-disable no-undef */
+require('@nomicfoundation/hardhat-chai-matchers');
+const { loadFixture } = require('@nomicfoundation/hardhat-network-helpers');
+const { expect } = require('chai');
+const { ethers, BigNumber } = require('ethers');
+
+const signPermit = require('./helpers/signPermit');
+
+const _1e18 = BigNumber.from('1000000000000000000');
```

Comment:
`BigNumber.from(1e18)`

Reply:
Unfortunately this doesn't work because it causes an overflow -- 1e18 is treated as a floating-point number so in this case i think we must use the string literal